src/Pages/EditPlaylist.js

```diff
@@ -45,18 +46,22 @@ const EditPlaylist = () => {
                         />)
 
   const handleChange = (e, type) => {
-    // console.log(`change ${type}`)
     dispatch(actions.updateTitleDescription(e.target.value, type));
+    return e.target.value.length > 0 ? (e.target.style.backgroundColor = 'white') : (e.target.style.backgroundColor = '#CECCCC')
   };
 
-  // only renders the create playlist button when all 5 games have been added
-  // const renderCreatePlaylistButton = createPlaylistGames.includes("Add a Game") ? (
-  //   'Add 5 Games'
-  // ) : (
-  //   <>
-  //     <button className="submit-playlist-button" onClick={ () => handleCreate() }>Create Playlist</button>
-  //   </> 
-  // )
+  const handleCreate = () => {
+    console.log(editPlaylistObj)
+  }
+
```

Comment:
I am assuming this is where the POST would go. We could send back the editPlaylistObj in the body of the POST request?

Reply:
I was just going to send the whole editPlaylistObj, but I can strip things out that you don’t need, if needed